### ISSUE TITLE
Replace: UATV

### ISSUE THREAD

**Issue by Smark223** [iptv-org/iptv#9448]
### Broken Link

https://ua-tv.cdn-02.cosmonova.net.ua/hls/ua-tv_ua.m3u8

### What happened to the stream?

Not loading

### Possible Replacement

https://freedom.cdn-01.cosmonova.net.ua/mobile-app/main/freedom/master.m3u8

### Notes

Streaming url for uatv.ua has changed. The lower resolution links in the ua playlist obviously no longer work either, so should be removed

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#report-a-broken-stream)

**Comment by LaneSh4d0w** [comment]
I believe the name change is made on purpose. Valid.

**Comment by Smark223** [comment]
Yes, I wondered if the channel name and tvg-id should be changed to 'Freedom'. They've used that on air name for a while now but the website is still uatv.ua
